src/routes/materi.js

```
@@ -0,0 +1,33 @@
+const express = require('express');
+const router = express.Router();
+const multer = require('multer');
+const materiController = require('../controllers/materi.controller');
+
+// Konfigurasi multer untuk file upload
+const storage = multer.diskStorage({
+  destination: function (req, file, cb) {
+    cb(null, 'uploads/'); // Pastikan folder uploads sudah ada
+  },
```

Biome (1.9.4)

[error] This function expression can be turned into an arrow function.

Function expressions that don't use this can be turned into arrow functions.
Safe fix: Use an arrow function instead.


(lint/complexity/useArrowFunction)